app/components/account/TokenAccountSection.tsx

```diff
@@ -157,9 +160,12 @@ function FungibleTokenMintAccountCard({
                 <tr>
                     <td>{mintInfo.mintAuthority === null ? 'Fixed Supply' : 'Current Supply'}</td>
                     <td className="text-lg-end">
-                        {normalizeTokenAmount(mintInfo.supply, mintInfo.decimals).toLocaleString('en-US', {
-                            maximumFractionDigits: 20,
-                        })}
+                        <span>
```

Comment:
<img width="547" alt="image" src="https://github.com/user-attachments/assets/cefafd53-287a-44fb-a506-05da2b05a270" />

I think that icon at the main card is not very well aligned. The simplest solution would be to adjust it's position via `transalte(-2px, -1px)`